src/graph/rg_matrix/rg_matrix.h

```diff
@@ -124,11 +124,11 @@ struct _RG_Matrix {
 
 GrB_Info RG_Matrix_new
 (
-    RG_Matrix *A,            // handle of matrix to create
```

Comment:
Why the change in indentation?

Reply:
From spaces to tabs as the format we use